### ISSUE TITLE
공동수강 진도 Machine Learning A to Z

### ISSUE THREAD

**Issue by kimshangyup** [kimshangyup/shaai#4]
진도 체크

**Comment by kimshangyup** [comment]
# 2주차
Data Preprocessing 부분
11. Missing Data
12. Categorical Data
13. Splitting the Dataset into the Training set and Test set

**Comment by kimshangyup** [comment]
# 1주차
1.Application of ML
2.why ML is future
3.Installing R/R Studio
4.Installing Python/Anaconda
(5.Meet your instructor)

6.Welcome to Part 1 - Data preprocessing
7.Get the dataset
8.Importing the Libraries
9.Importing the Dataset
10.Summary of OOP

**Comment by kimshangyup** [comment]
# 2주차
Data Preprocessing 부분
11. Missing Data
12. Categorical Data
13. Splitting the Dataset into the Training set and Test set
